package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "alanlauer.com",
+  "version": "1.0.0",
+  "private": true,
+  "description": "alanlauer.com",
+  "author": "Brometheus",
+  "keywords": [
+    "gatsby"
+  ],
+  "scripts": {
+    "develop": "gatsby develop",
+    "start": "gatsby develop",
+    "build": "gatsby build",
+    "serve": "gatsby serve",
+    "clean": "gatsby clean"
+  },
+  "dependencies": {
```

Comment:
can you add `gh-pages` dependency? we will use that in a new "deploy" command on the scripts list above. 

the script for deploying with gh-pages is something like: 
```
gatsby build --prefix-paths && gh-pages -d public
```
but i'm not certain you'll need the pre-fix paths argument. i just had to in the past so left it just in case.